panel/src/config/api.js

```diff
@@ -48,16 +49,13 @@ export default {
 					}
 
 					Vue.$api.requests.push(requestId);
-				},
-				onSuccess: () => {
-					clearInterval(Vue.$api.ping);
-					Vue.$api.ping = setInterval(Vue.$api.auth.user, 5 * 60 * 1000);
```

Comment:
We had this code so that the ping can be skipped while the user is active. So the ping would only be sent if there was no activity in the last five minutes. I think this still makes sense.